scout/demo/resources/__init__.py

```diff
@@ -1,62 +1,34 @@
-import pkg_resources
+import importlib_resources
 
-###### Files ######
-
-# Gene sources:
-resources_folder = "demo/resources"
-hgnc_file = "demo/resources/hgnc_reduced_set.txt"
-exac_file = "demo/resources/forweb_cleaned_exac_r03_march16_z_data_pLI_reduced.txt"
-transcripts37_file = "demo/resources/ensembl_transcripts_37_reduced.txt"
-genes37_file = "demo/resources/ensembl_genes_37_reduced.txt"
-exons37_file = "demo/resources/ensembl_exons_37_reduced.txt"
-transcripts38_file = "demo/resources/ensembl_transcripts_38_reduced.txt"
-genes38_file = "demo/resources/ensembl_genes_38_reduced.txt"
-exons38_file = "demo/resources/ensembl_exons_38_reduced.txt"
-
-# OMIM resources:
-mim2gene_file = "demo/resources/mim2gene_reduced.txt"
-genemap2_file = "demo/resources/genemap2_reduced.txt"
-
-# Hpo resources
-genes_to_phenotype_to_diseases_file = "demo/resources/genes_to_phenotype_reduced.txt"
-hpoterms_file = "demo/resources/reduced.hpo.obo"
-hpo_phenotype_to_genes_to_diseases_file = "demo/resources/phenotype_to_genes_reduced.txt"
-
-hpo_terms_def_file = "demo/resources/hpo_terms.csv"
-
-# Additional resources
-madeline_file = "demo/madeline.xml"
+BASE_PATH = importlib_resources.files("scout")
```

Comment:
```suggestion
BASE_PATH = importlib_resources.files("scout").joinpath("demo").resolve()
```

Reply:
I have fixed by joining the path like this instead:

```
from importlib_resources import files

BASE_PATH = "scout.demo"

ped_path = str(files(BASE_PATH).joinpath("643594.ped"))
```

Which should work on any platform!